CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 Please enumerate all user-facing changes using format `<githib issue/pr number>: <short description>`, with changes ordered in reverse chronological order.
 
 ## SNAPSHOT
+## 0.13.0
+
+* [#544](https://github.com/kroxylicious/kroxylicious-junit5-extension/pull/544): Replace Keytool certificate generator by netty certificate builder
+
```

Comment:
Probably worth adding a Changes, deprecations and removals section telling users to use the new API.

Reply:
It is below that in the file